src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -56,25 +56,6 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
         buildResult.output.contains("name is a required docker configuration item.")
     }
 
-    def 'fail with missing dockerfile'() {
```

Comment:
I removed the test since we're no longer the ones throwing the error in this case, docker is. We can still have this test, but it would only be testing the failure of the task and not the error that we used to throw.